hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/TokenTupleUtils.java

```diff
@@ -60,7 +61,7 @@ public static Tuple expiryTupleFor(@NonNull final Token token) {
         return Tuple.of(
                 token.expirationSecond(),
                 headlongAddressOf(token.autoRenewAccountIdOrElse(ZERO_ACCOUNT_ID)),
-                token.autoRenewSeconds());
+                (token.autoRenewSeconds() > 0 ? token.autoRenewSeconds() : 0));
```

Comment:
This expression would be different from just `token.autoRenewSeconds()` **only** if `token.autoRenewSeconds() < 0`.  

Is that possible? 🤔

Reply:
Yes, unfortunately. The test was failing because autoRenewSeconds is an unsigned integer and it was a negative vaue.